### DISCUSSION TITLE
Special Variables

### DISCUSSION THREAD

**Discussion by dannyaguero** [TurboWarp/packager#134] [Ideas]
Similar to HTMLifier, can there be an option for cloud variables where giving cloud variables with certain names special behaviours? (ex. open link or eval) It would really be useful for things that specifically use TurboWarp features that also use special things (like links or javascript executables) (This is what i'm talking about for those who don't know: https://sheeptester.github.io/htmlifier/index.html#:~:text=Give%20cloud%20variables%20with%20certain%20names%20special%20behaviours.)

**Comment by GarboMuffin** [comment]
Follow #48
